src/structlog/_frames.py

```diff
@@ -6,32 +6,39 @@
 import traceback
 
 from io import StringIO
+from types import FrameType
+from typing import List, Optional, Tuple
 
+from .types import ExcInfo
```

Comment:
I've had problems with mypy in the past with modules named `types` -- good to see they've fixed some of that! (it used to get them confused in weird ways with the stdlib `types` module)

Reply:
😬

Reply:
Ah nice Emacs' checker just started yelling at me at user-defined types modules are not permitted. I guess typing it is?